### PR TITLE
fix for issue #297, adding the path for the ldap auth plugin on CentOS 7

### LINKED ISSUE #297: openvpn-auth-ldap.so plugin not being loaded in CentOS 7
I am deploying an openvpn server on CentOS 7, and openvpn fails to start due to an invalid config. I've traced it to this line (https://github.com/voxpupuli/puppet-openvpn/blob/master/manifests/params.pp#L33) unsetting the lib file. This writes out an invalid plugin line in the config causing the following error to appear.

`
Thu Oct 11 10:49:30 2018 us=958804 PLUGIN_INIT: could not load plugin shared object /etc/openvpn/vpnserver/auth/ldap.conf: /etc/openvpn/vpnserver/auth/ldap.conf: invalid ELF header: No such file or directory (errno=2)
`

The config looks like this:
`
plugin  "/etc/openvpn/vpnserver/auth/ldap.conf"
`

when it should look like this:
`
plugin /usr/lib64/openvpn/plugin/lib/openvpn-auth-ldap.so "/etc/openvpn/vpnserver/auth/ldap.conf"
`

Error is in the current commit, as well as the module on puppetforge which I'm currently trying to use.

### PR DESCRIPTION
**Pull request by kindred**
#### Pull Request (PR) description
A fix for the malformed config file on CentOS 7 when using the LDAP auth plugin.

#### This Pull Request (PR) fixes the following issues
Fixes #297

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -30,7 +30,7 @@
       case $facts['os']['release']['major'] {
         '7': {
           $additional_packages = ['easy-rsa']
```
Comment (alexjfisher) [inline manifests/params.pp]: Should this now also be `$additional_packages = ['easy-rsa','openvpn-auth-ldap']` for EL7 as well as EL6?
(If these parameters are now the same on EL6 and EL7, they can also be moved out of this case statement).

Comment (alexjfisher) [reply manifests/params.pp]: Did EL7 packages become available some time *after* https://github.com/voxpupuli/puppet-openvpn/pull/215 ?

**Comment by Dan33l** [comment]
Looks related with issue #172

**Comment by alexjfisher** [comment]
openvpn-auth-ldap is definately available in [epel 7](https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/o/openvpn-auth-ldap-2.0.3-15.el7.x86_64.rpm) these days.  No idea since when though.

File: manifests/params.pp (hunk 1 above)
Comment (kindred) [reply manifests/params.pp]: It looks like the openvpn-auth-ldap package is now available in the repo for EL7. Will amend the PR.

**Comment by Dan33l** [comment]
Oups ? why closed ?

**Pull request closed without merging**